tests/qm-sanity-test/test.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+source ../e2e/lib/utils
+
+# Verify qm is up and running
+check_qm_is_up(){
```

Comment:
In 2nd thought i think eacho one of the function in a test itself I think you could split 
tests/qm-sanity-test/main.fmf

Into smaller parts like 
/check-qm-up
test: source test.sh; check_qm_is_up

each one of the tests will have its own id
so in that case this script has 6 tests, if one of them is failing, all of the 6 are marked as failed

Reply:
@Yarboa thank you so much. I split the test into 6 tests, which seems more reasonable.

Reply:
yes I agree it will add little bit granularity